### PR TITLE
DALA 344 Fixed Euro-Sign alignment in DataTable

### PR DESCRIPTION
**Pull request by MarcMDfine**
# Pull Request \<Title>
`<Description here>`
## Things to do during Peer Review
Please check all boxes before the Pull Request is merged. In case you skip a box, describe in the PRs description (that means: here) why the check is skipped.
- [x] The Github Actions (including Sonarqube Gateway and Lint Checks) are green. This is enforced by Github. 
- [x] A peer-review has been executed
  - [x] The code has been manually inspected by someone who did not implement the feature
- [x] The PR actually implements what is described in the JIRA-Issue
- [x] ~At least one E2E Test exists testing the new feature~
- [x] ~Documentation is updated as required~
- [x] ~The automated deployment is updated if required~
- [x] The new version is deployed to the dev server using this branch
  - [x] It's verified that this version actually is the one deployed (check actuator/info for branch name and commit id!)
  - [x] The new feature is manually used/tested/observed on dev server
  - [x] All implemented Social Logins have been tested manually in the UI
- [x] ~There is at least one picture for each story, which was created before coding has started~
- [x] If any work on the UI is to be merged, those changes were also documented in the Figma 
- [x] The local Dev stack still works: execute `startDevelopmentStack.sh`, npm serve, npm teste2e and execute Cypress Tests locally

### CODE REVIEW

File: dataland-frontend/src/components/resources/taxonomy/search/EuTaxoSearchResults.vue [hunk 1]
```diff
@@ -131,4 +132,8 @@ export default {
 #search-result-taxonomy th {
   background: white;
 }
+
+.d-justify-content-end-inner > div {
+  justify-content: end;
+}
```
Comment (emanueldfine) [inline dataland-frontend/src/components/resources/taxonomy/search/EuTaxoSearchResults.vue:138]: Ich glaube wir hatten mal gesagt, dass wir custom styles scopen wollen.
Ich frage nochmal im Chat nach und schließe die Conversation ansonsten. Stephan müsste das noch wissen.

Comment (MarcMDfine) [reply dataland-frontend/src/components/resources/taxonomy/search/EuTaxoSearchResults.vue:138]: Wenn ich die Klasse in ein <style scoped> packe, dann klappt das ganze aber nicht mehr. Ich glaube das liegt daran, dass die Klasse nicht in der Komponente, sondern in der PrimeVue Column-Componente genutzt wird

File: dataland-frontend/src/components/resources/taxonomy/search/EuTaxoSearchResults.vue [hunk 2]
```diff
@@ -53,6 +53,7 @@
           <Column
             field="companyInformation.marketCap"
             header="MARKET CAP"
+            headerClass="d-justify-content-end-inner"
             :sortable="true"
             class="d-bg-white w-1 text-right"
           >
```
Comment (emanueldfine) [inline dataland-frontend/src/components/resources/taxonomy/search/EuTaxoSearchResults.vue:59]: Off-Topic:

Gezwungenermaßen testen wir ja die Social Logins nicht im e2e Test.
Was hälst du von der Idee, deren Selektierbarkeit zumindest zu testen?
Also einfach ob die Buttons dafür da sind.

File: dataland-frontend/src/components/resources/taxonomy/search/EuTaxoSearchResults.vue (hunk 1 above)
Comment (emanueldfine) [reply dataland-frontend/src/components/resources/taxonomy/search/EuTaxoSearchResults.vue:138]: Wie im Chat besprochen über ID-Zuordnung von anderen Komponenten fern halten.

Comment (MarcMDfine) [reply dataland-frontend/src/components/resources/taxonomy/search/EuTaxoSearchResults.vue]: Done

**Pull request merged**